Tests/ClientTests/Frontend/Home/RecentlySaved/RecentlySavedDataAdaptorTests.swift

```diff
@@ -13,15 +13,15 @@ class RecentlySavedDataAdaptorTests: XCTestCase {
     var mockSiteImageHelper: SiteImageHelperMock!
     var mockReadingList: ReadingListMock!
     var mockBookmarksHandler: BookmarksHandlerMock!
-    var spyNotificationCenter: SpyNotificationCenter!
+    var mockNotificationCenter: MockNotificationCenter!
     var mockDelegate: RecentlySavedDelegateMock?
 
     override func setUp() {
         super.setUp()
         mockSiteImageHelper = SiteImageHelperMock()
         mockReadingList = ReadingListMock()
         mockBookmarksHandler = BookmarksHandlerMock()
-        spyNotificationCenter = SpyNotificationCenter()
+        mockNotificationCenter = MockNotificationCenter()
```

Comment:
Ah nice! That was my next question if we should replace that Spy haha. Thanks!

Reply:
I think the spy is still fine to use for testing classes that post notifications but it's more difficult to use for testing listeners without adding async code to the tests which is why I couldn't really make use of it here.